app/src/frontend/map-app.tsx

```diff
@@ -56,6 +58,7 @@ class MapApp extends React.Component<MapAppProps, MapAppState> {
 
     componentDidMount() {
         this.fetchLatestRevision();
+        this.fetchBuildingData();
```

Comment:
This needs to happen when a building page (e.g. `/view/age/233771`) is navigated to from another place in the app. Previously there were no links to building pages in the app, so the data was either loaded in `frontendRoute.tsx` as part of server-side rendering, or it was loaded upon selecting a building on the map.
Now, with the addition of global edit history screen, the edit records link to building pages, and so there needs to be a mechanism to load the data when a component is mounted but there is no pre-loaded data from SSR.